java-ast/tests/pom.xml

```diff
@@ -41,6 +41,12 @@
             <artifactId>windup-exec</artifactId>
             <classifier>forge-addon</classifier>
             <scope>test</scope>
+        </dependency>    
+        <dependency>
+            <groupId>org.eclipse.tycho</groupId>
+            <artifactId>org.eclipse.jdt.core</artifactId>
+            <version>3.10.0.v20140604-1726</version>
+            <scope>test</scope>
```

Comment:
Maven 3.2.5 but that should not be a cause, I am switching between 3.3.9 and 3.2.5 regularly